cmd/pipecd/ops.go

```diff
@@ -79,6 +80,35 @@ func (s *ops) run(ctx context.Context, t cli.Telemetry) error {
 		return err
 	}
 
+	// Prepare sql database.
+	if cfg.Datastore.Type == model.DataStoreMySQL {
+		mysqlEnsurer := mysqlensurer.NewMySQLEnsurer(
+			cfg.Datastore.MySQLConfig.URL,
+			cfg.Datastore.MySQLConfig.Database,
+			cfg.Datastore.MySQLConfig.UsernameFile,
+			cfg.Datastore.MySQLConfig.PasswordFile,
+			t.Logger,
+		)
+		defer func() {
+			// Close connection held by the client.
+			if err := mysqlEnsurer.Close(); err != nil {
```

Comment:
Thanks for moving it. Sorry but tell me a little bit more about is there a reason why you don't want to call `mysqlEnsurer.connect()` in the `mysqlensurer.NewMySQLEnsurer` method? If nil was populated to client, I feel like panic can occur.